lib/basedriver/commands/find.js

```diff
@@ -341,35 +359,74 @@ helpers.getScreenshotForImageFind = async function (screenWidth, screenHeight) {
   // are two potential types of mismatch: aspect ratio mismatch and scale
   // mismatch. we need to detect and fix both
 
-  const screenAR = screenWidth / screenHeight;
-  const shotAR = shotWidth / shotHeight;
-
-  if (screenAR === shotAR) {
-    log.info('Screenshot aspect ratio matched screen aspect ratio');
-  } else {
-    log.warn(`When trying to find an element, determined that the screen ` +
-             `aspect ratio and screenshot aspect ratio are different. Screen ` +
-             `is ${screenWidth}x${screenHeight} whereas screenshot is ` +
-             `${shotWidth}x${shotHeight}.`);
-    shotWidth = shotWidth / (shotAR / screenAR);
-    log.warn(`Resizing screenshot to ${shotWidth}x${shotHeight} to match ` +
-             `screen aspect ratio so that image element coordinates have a ` +
-             `greater chance of being correct.`);
-    imgObj = imgObj.resize(shotWidth, shotHeight);
-  }
-
-  // now we know the aspect ratios match, but there might still be a scale
-  // mismatch, so just resize based on the screen dimensions
-  if (screenWidth !== shotWidth) {
+  // const screenAR = screenWidth / screenHeight;
```

Comment:
Will remove here if we do not need these lines

Reply:
yep, I vote for removing the commented stuff